inbox/mailsync/backends/imap/common.py

```diff
@@ -204,6 +208,7 @@ def remove_deleted_uids(account_id, folder_id, uids):
                     ImapUid.folder_id == folder_id,
                     ImapUid.msg_uid == uid,
                 )
+                .with_hint("FORCE INDEX (ix_imapuid_account_id_folder_id_msg_uid_desc)")
```

Comment:
Here this is not covering index, because we retrieve other columns, but it does not make things slower, it just switches from the old to the new index.